### DISCUSSION TITLE
使用katex渲染输出结果

### DISCUSSION THREAD

**Discussion by KagaJiankui** [OleehyO/TexTeller#44] [Ideas]
我使用的latex引擎是tectonic, 对于不准备安装TeXLive的用户来说, 能否设法在web端使用katex渲染输出结果?

**Comment by OleehyO** [comment]
模型输出的有些latex指令katex不支持，所以才使用xelatex在web端进行渲染。
如果一定要用katex渲染输出结果，可以考虑修改[web.py](https://github.com/OleehyO/TexTeller/blob/main/src/web.py)

**Comment by OleehyO** [comment]
更新了，现在已经支持了katex了。
